### ISSUE TITLE
Replace Round markers by Line-Detection or Laser-Bed edges

### ISSUE THREAD

**Issue by t-oster** [t-oster/VisiCam#16]
It is not very pleasant to add markers to the laser-bed. Since most laser-beds are rectangular, it may be possible to just run a line-detection algorithm on the image and find the outer-most and most-perpendicular rectangle and use that for the homography. This way nearly any Lasercutter can be used without preparation, just place a camera above it and maybe mark the areas where the algorithm shall search for the lines.

However, since I am not that up to date with OpenCV, and I do not have a VisiCam setup right now, I won't have time to develop it. So if anyone wants to improve VisiCam, this may be an idea.

**Comment by mgmax** [comment]
I don't think this will be very robust - even the round markers sometimes cause trouble under changing lighting conditions. But if it works and you can switch back to the round markers, why not?
